### ISSUE TITLE
Add calender/scheduling feature into Discussion threads

### ISSUE THREAD

**Issue by RuubixO** [codebuddies/codebuddies#1123]
Do you want to request a feature or report a bug?
Feature.

What is the current behavior?
Discussion is made for a proposed CB Hangout, but there's no option for a quick reference to everyone's availability.

What is the expected behavior?

1. Once I've made a new Discussions post/thread for a Hangout Proposal, 

2. A Calendar widget should appear on the left sidebar menu underneath "PEOPLE IN THIS DISCUSSION" on the left of the page. 

3. The calendar should be formatted to show the current month with highlighted days that people have selected availability. 

         - Clicking the calendar brings up a google calendar with a more detailed selection of what availability each person has offered. This should also be the place for people to add/edit their own availability.

Which browser / OS are affected by this issue? (An *isolated* way to reproduce the bug)
I use Fedora 29.

**Comment by RuubixO** [comment]
I just realized these two might be close enough to be combined: https://github.com/codebuddies/codebuddies/issues/1118

**Comment by billglover** [comment]
In the use case that you describe, where is this discussion happening? The forum, on Slack or on the CodeBuddies site? After a quick look around it looks as though you are describing the [discussion](https://codebuddies.org/discussion/i5jdgkKh5Mg4jGevm). There is no reason the functionality shouldn't apply to all three.

This feature sounds like it is aligned with the primary goal of [CodeBuddies](https://medium.com/codebuddies/codebuddies-in-2018-a-project-status-letter-to-the-community-7f8c2c34e950#8404) and it looks to help address the following explicit challenges:
- figure out timezones - by showing availability
- invite participants - by making it easy to invite people who have been involved in the discussion

**Comment by RuubixO** [comment]
Admittedly, I'm making the assumption that the discussion is centralized on the post within the codebuddies.org/discussions page. Otherwise, the discussion page itself doesn't serve much of a purpose for me. I have noticed that when I did make the discussion page, it did not generate a lot of traffic/views. Or at least, it's not clear if slack's higher activity de-incentivizes members from moving onto the website.

**Comment by RuubixO** [comment]
@billglover It would be good to have a way for people to view a given thread from either the post on "Discussions" OR on slack. I noticed you have an app which can add github "issue" comments directly from there. Having a similar feature would be convenient for this and keep the flow organized by the thread on the website, but also organic, in the way active slack users can respond directly from the channel (that was added by the discussion-post when the original poster created it).

**Comment by RuubixO** [comment]
I agree that giving flexibility for participants to create and participate in Discussion posts both from slack and from GitHub would be ideal.

**Comment by billglover** [comment]
I notice the original discussions post has been removed. Are you able to provide a link to a discussion regarding hang-out scheduling to help illustrate the context. I've tried to illustrate what I think is being requested below.

As I understand it currently:
- a person visits codebuddies.org/discussions and proposes a hangout
- other people comment on the proposal
- the initial author decides to schedule the hang-out

New functionality requested:
- the initial author can see availability of all those who participated in the discussion
- the initial author can use the information around availability to schedule the hang-out

I'd like to treat cross-posting of conversation threads between platforms as a separate topic for now.
